### PR TITLE
Update on-premises-network-connections.md

### PR DESCRIPTION
**Pull request by juqassis**
There appears to be a typo where I made my edit on the doc under the "provisioning" section. It's a very minor correction, but threw me off when I was reading it, so figured i'd bring some attention to it. Thanks!

### CODE REVIEW

**Comment by PRMerger13** [comment]
@juqassis : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger6** [comment]
@ErikjeMS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ErikjeMS** [comment]
#sign-off

File: windows-365/enterprise/on-premises-network-connections.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ There are two kinds of OPNCs based on their join type. Both let you manage traff
 
 ## Provisioning
 
-When a Cloud PC is provisioned, the information in the OPNC is used by the provisioning policy to provision the Cloud PC the Azure subnet. The information required in an OPNC includes:
+When a Cloud PC is provisioned, the information in the OPNC is used by the provisioning policy to provision the Cloud PC in the Azure subnet. The information required in an OPNC includes:
 
 - **Network details**: The Azure subscription, resource group, virtual network, and subnet that the Cloud PC will be associated with. When a provisioning policy runs, it creates a Cloud PC in the Microsoft hosted Azure subscription. To connect to a customers on-premises network, a virtual network interface card (vNic) is injected into a customer-provided Azure virtual network (vNet). To create this vNic, Windows 365 needs sufficient access to an Azure subscription.
```
Comment (JohanFreelancer9) [inline windows-365/enterprise/on-premises-network-connections.md:50]:
```suggestion
- **Network details**: The Azure subscription, resource group, virtual network, and subnet that the Cloud PC will be associated with. When a provisioning policy runs, it creates a Cloud PC in the Microsoft hosted Azure subscription. To connect to a customer's on-premises network, a virtual network interface card (vNic) is injected into a customer-provided Azure virtual network (vNet). To create this vNic, Windows 365 needs sufficient access to an Azure subscription.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review - @juqassis, kindly commit the additional suggested change. Many thanks.

**Comment by PRMerger17** [comment]
@ErikjeMS : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by ErikjeMS** [comment]
The filename has changed for this article. To expedite, I'll make the changes in another PR 7551 and close this PR. Thanks!

**Pull request closed without merging**